### PR TITLE
Add employment history for 5490

### PR DESCRIPTION
**Pull request by jbalboni**
Related to [721](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/721)

screen shot 2017-03-06 at 4 04 04 pm
screen shot 2017-03-06 at 4 04 18 pm

### CODE REVIEW

File: src/js/edu-benefits/5490/config/form.js [hunk 1]
```diff
@@ -69,9 +71,40 @@ const formConfig = {
...
+                itemName: 'Employment Period',
+                viewField: EmploymentPeriodView,
+                hideTitle: true
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              nonMilitaryJobs: _.unset('items.properties.postMilitaryJob', nonMilitaryJobs)
```
Comment (jbalboni) [inline src/js/edu-benefits/5490/config/form.js]: Going to remove this from the schema, I think.

Comment (cvalarida) [inline src/js/edu-benefits/5490/config/form.js]: Makes sense if you can't add `postMilitaryJob` anywhere. I take it it's not being used elsewhere? Where does it come from?

File: test/edu-benefits/5490/config/employmentHistory.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          onSubmit={onSubmit}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = findDOMNode(form);
+    submitForm(form);
+
+    expect(Array.from(formDOM.querySelectorAll('.usa-input-error'))).to.be.empty;
+
+    submitForm(form);
```
Comment (cvalarida) [inline test/edu-benefits/5490/config/employmentHistory.unit.spec.jsx]: This second call probably isn't necessary if there are no errors.

**Review by cvalarida — APPROVED** [review]
I had a question and comment, but it looks good.

File: src/js/edu-benefits/5490/config/form.js (hunk 1 above)
Comment (jbalboni) [inline src/js/edu-benefits/5490/config/form.js]: After talking with Alex, we realized it's from the 1990 form. This is using the same definition. It looks awkward to remove it on the backend, so I may leave it for now.

**Pull request merged**